Explorer/Assets/DCL/MapRenderer/ConsumerUtils/MapRenderImage.cs

```diff
@@ -177,7 +177,7 @@ private void ProcessHover(PointerEventData eventData)
                 {
                     if (previousMarker != null)
                     {
-                        previousMarker.SetIconOutline(false);
+                        previousMarker.AnimateDeselectionAsync().Forget();

```

Comment:
Did we remove the outline shown and hidden when hovering?

Reply:
Nope, but its now part of the AnimateSelection/Deselection